src/Data/Food/Builder/Recipe.elm

```diff
@@ -165,7 +159,7 @@ compute db =
 
                     transformImpacts =
                         transform
-                            |> Maybe.map (computeProcessImpacts db.impacts >> List.singleton >> updateImpacts)
+                            |> Maybe.map (computeProcessImpacts >> List.singleton >> Impact.sumImpacts db.impacts)
```

Comment:
So we're creating a list with a single item which we sum. I'm pretty confident we could skip the singleton + sum stuff here.